src/components/notifications/Pushdeer.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="mb-3">
+        <label for="pushdeer-key" class="form-label">Pushdeer Key</label>
```

Comment:
This should use the translations. See https://github.com/louislam/uptime-kuma/blob/c9deea9fdf8dbe0a0f96061ea75d996b47f0ce07/src/components/notifications/Discord.vue#L3 for an example